src/SW_Carbon.c

```diff
@@ -272,8 +272,8 @@ void SW_CBN_init_run(VegType VegProd_veg[], SW_MODEL* SW_Model,
     if (SW_Carbon->use_bio_mult) {
       ForEachVegType(k) {
         VegProd_veg[k].co2_multipliers[BIO_INDEX][year] =
-                                              VegProd_veg[k].co2_bio_coeff1
-                                              * pow(ppm, VegProd_veg[k].co2_bio_coeff2);
+                            VegProd_veg[k].co2_bio_coeff1

```

Comment:
please only do white-space adjustments when actually something in the code changes - if at all